.devcontainer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG IMAGE_TYPE=latest-cpu
+FROM tfaddons/dev_container:$IMAGE_TYPE
+
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Options for common package install script - SHA updated on release
+ARG INSTALL_ZSH="false"
+ARG UPGRADE_PACKAGES="false"
+ARG COMMON_SCRIPT_SOURCE="https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/script-library/common-debian.sh"
+ARG COMMON_SCRIPT_SHA="dev-mode"
+
+# Configure apt and install packages
+RUN apt-get update \
```

Comment:
@Chuxel Some section like this one was inspired by [your template](https://github.com/microsoft/vscode-dev-containers/blob/main/container-templates/dockerfile/.devcontainer/Dockerfile) that has no COPYRIGHT.
So I suppose it is fine and in the scope of the template to derive from that without using the MS copyright.